docker-compose.test.yml

```
@@ -0,0 +1,45 @@
+---
+# Compose file for test runs
+version: "3.7"
+services:
+  db:
+    image: datarttu/sujuikodb:latest
+    build: .
+    restart: unless-stopped
```

```suggestion
    restart: "no"
```
What if we aim to make this runnable in a test script that could be run both locally and in CI?

---

This is probably OK, I'm not quite familiar with docker-compose restart options though.